src/queue-manager.js

```diff
@@ -1,7 +1,69 @@
 let SQSConsumer = require('sqs-consumer');
-let debugModule = require('debug');
+let debug = require('debug')('cloud-mirror:queue');
 let assert = require('assert');
 let _ = require('lodash');
+let slugid = require('slugid');
+
+
+/**
```

Comment:
I moved this out of the main start up so that we're no longer creating each queue the same number of times we have each consumer start.

Longer term, the goal is to move both SQS queue and S3 bucket creation and setup into a completely seperate bit of code.